crates/evm/core/src/backend/mod.rs

```diff
@@ -1263,6 +1273,53 @@ impl DatabaseExt for Backend {
         None
     }
 
+    /// Loads the account allocs from the given `allocs` map into the passed [JournaledState].
+    ///
+    /// Returns [Ok] if all accounts were successfully inserted into the journal, [Err] otherwise.
+    fn load_allocs(
+        &mut self,
+        allocs: &HashMap<Address, GenesisAccount>,
+        journaled_state: &mut JournaledState,
+    ) -> Result<(), DatabaseError> {
+        // Loop through all of the allocs defined in the map and commit them to the journal.
+        allocs.iter().try_for_each(|(addr, acc)| {
```

Comment:
I'd prefer a simple loop here

Reply:
done in https://github.com/foundry-rs/foundry/pull/6207/commits/1292c7b9856b1391d06a456398ab2417f2831134